src/App.js

```diff
@@ -1,13 +1,54 @@
 import React from "react";
-
+import TouristInfoCards from "./TouristInfoCards";
 import Bookings from "./Bookings";
+import Restaurant from "./Restaurant";
+import Heading from "./Heading";
+import Footer from "./Footer";
 import "./App.css";
 
+const Glasgow = {
+  src:
+    "https://peoplemakeglasgow.com/assets/general/_1200x600_crop_center-center_none/Eco-City-Break-Queens-Park-Viewpoint-Header-Block.jpg",
+  cardTitle: "Glasgow",
+  cardText: " sdgssg",
+  href: "https://peoplemakeglasgow.com/"
+};
+
+const Manchester = {
+  src:
+    "https://eu-assets.simpleview-europe.com/manchester2016/imageresizer/?image=%2Fdbimgs%2FManOne-03%284%29.jpg&action=Background_Overlay",
+  cardTitle: "Manchester",
+  cardText:
+    " Manchester is the only UK city to feature in Lonely Planet's Best in Travel 2023 list and the only UK city in National Geographic's influential ‘Best of the World’ list which annually sets out 25 of the must-see places to visit around the globe.",
+  href: "https://www.visitmanchester.com/"
+};
+
+const London = {
```

Comment:
I like how you are constructing the data here! But I would still use a lower case for all js variables like this. Even though London is a name of a place etc!